spectrometer.py

```diff
@@ -74,17 +74,15 @@ def findAperture(pix,middleX,middleY):
 
 # draw aperture onto image
 def drawAperture(aperture,draw):
-	draw.setink("#000");
-	draw.line((aperture['x'],aperture['y']-aperture['h']/2,aperture['x'],aperture['y']+aperture['h']/2))
+	draw.line((aperture['x'],aperture['y']-aperture['h']/2,aperture['x'],aperture['y']+aperture['h']/2),fill="#000")
 
 #draw scan line
 def drawScanLine(aperture,spectrumAngle,draw):
 	xd=aperture['x']
 	h=aperture['h']/2
 	y0=math.tan(spectrumAngle)*xd+aperture['y']
-	draw.setink("#888");
-	draw.line((0,y0-h,aperture['x'],aperture['y']-h))
-	draw.line((0,y0+h,aperture['x'],aperture['y']+h))
+	draw.line((0,y0-h,aperture['x'],aperture['y']-h),fill="#888")
```

Comment:
Icing on cake, but we could make these colors injectable to make it more flexible later.

Not a blocker.